code/src/index.css

```diff
@@ -1,13 +1,225 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
   font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
     "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background: #000;
+  color: #fff;
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+main {
+  max-width: 1200px;
+  margin: 0 auto;
+  width: 100%;
+  padding: 0 50px;
+}
+
+/* HEADER */
+
+header {
+  font-size: 20px;
+  font-weight: bold;
+  padding-right: 20px;
+  min-height: 6vh;
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+}
+
+header form {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+header label {
+  margin-right: 15px;
+}
+
+header select {
+  height: 30px;
+  width: 200px;
+  font-family: inherit;
+  font-size: inherit;
+  font-weight: inherit;
+}
+
+/* MOVIE LIST */
+
+.popular-page {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.popular-page a {
+  width: 25%;
+  position: relative;
+  color: #fff;
+  text-decoration: none;
+}
+
+.popular-page a img {
+  width: 100%;
+}
+
+.popular-page .details {
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  background: rgba(0,0,0,0.75);
+  display: none;
+  padding: 20px;
+}
+
+.popular-page .details h1 {
+  margin-top: auto;
+  margin-bottom: 0;
+  font-size: 2em;
+  margin-block-start: 0.67em;
+  margin-block-end: 0.67em;
+}
+
+.popular-page a:hover .details {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+  justify-content: flex-end;
+}
+
+/* MOVIE DETAILS */
+
+.detail-page {
+  position: relative;
+}
+
+.detail-page .back-link {
+  position: absolute;
+  display: inline-flex;
+  color: #fff;
+  align-items: center;
+  font-weight: bold;
+  text-decoration: none;
+  left: 50px;
+  top: 50px;
+}
+
+.detail-page .back-link svg {
```

Comment:
I should have used this approach instead of making the entire back button. This is great :)